shark/src/test/java/shark/HeapAnalyzerTest.kt

```diff
@@ -155,4 +156,16 @@ class HeapAnalyzerTest {
     val leaks = hprofFile.checkForLeaks<HeapAnalysisSuccess>()
     assertThat(leaks.applicationLeaks).hasSize(1)
   }
+
+  @Test fun failsOnObfuscatedCode() {
+    hprofFile.writeMultipleActivityLeaks(0)
+
+    val analysis = hprofFile.checkForLeaks<HeapAnalysisFailure>(
```

Comment:
nit: make it obvious that we're passing a null proguard mapping (ie add that arg)?